keras_cv/layers/preprocessing/random_color_jitter_test.py

```diff
@@ -22,9 +21,9 @@
 class RandomColorJitterTest(TestCase):
     # Test 1: Check input and output shape. It should match.
     def test_return_shapes(self):
-        batch_input = np.ones((2, 512, 512, 3))
-        non_square_batch_input = np.ones((2, 1024, 512, 3))
-        unbatch_input = np.ones((512, 512, 3))
+        batch_input = tf.ones((2, 512, 512, 3))
```

Comment:
why do we need to change them to `tf.`?  I think `np.` should also work since we allow for numpy args with the variable set in base layer

Reply:
Same in many tests

Reply:
done